src/xdsl/xdsl_opt_main.py

```diff
@@ -211,7 +246,6 @@ def setup_pipeline(self):
         pipeline = [
             str(item) for item in self.args.passes.split(',') if len(item) > 0
         ]
-
```

Comment:
Could we remove those changes from this PR? This makes the blame quite unreadable.